Frameworks/include/UIViewInternal.h

```diff
@@ -33,6 +33,27 @@
 @class WXFrameworkElement;
 @class WUXIPointerRoutedEventArgs;
 
+// Round subpixel values to be able to perform per-pixel UI placement/calculations
+inline float doPixelRound(float f) {
```

Comment:
the image proxy still exists, so this probably won't help them

---
In reply to: [100639129](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r100639129) [](ancestors = 100639129,100638812,100634412)